src/main/java/org/jboss/tools/intellij/openshift/ui/helm/InstallPanel.java

```diff
@@ -120,6 +123,11 @@ private void initComponents() {
     releaseNameText.addKeyListener(onKeyPressed(installButton));
     add(releaseNameText, "spanx 2, pushx, growx, wrap");
 
+    add(new JBLabel("Active Project:"),"skip");
```

Comment:
Is it possible tochange that label given if the cluster is openshift (project) or kubernetes(namespace) ?

Reply:
replaced by
```java
add(new JBLabel("Active " + (odo.isOpenShift()? "project:" : "namespace:")),"skip");
```